<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AiCopilotDP - AI Studio</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lbl_title">
      <property name="text">
       <string>üõ†Ô∏è AI Studio - Tool Manager</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="styleSheet">
       <string>font-size: 18pt; font-weight: bold; padding: 10px;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="grp_tools">
      <property name="title">
       <string>Available Tools</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QListWidget" name="lst_tools">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="btn_launch">
           <property name="text">
            <string>‚ñ∂Ô∏è Launch Tool</string>
           </property>
           <property name="minimumHeight">
            <number>50</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_refresh">
           <property name="text">
            <string>üîÑ Refresh List</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_add_tool">
           <property name="text">
            <string>‚ûï Add Tool</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_help">
           <property name="text">
            <string>üìñ Help</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_settings">
           <property name="text">
            <string>‚öôÔ∏è Settings</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="grp_info">
      <property name="title">
       <string>Tool Information</string>
      </property>
      <property name="maximumHeight">
       <number>150</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTextEdit" name="txt_tool_info">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd"&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name="qrichtext" content="1" /&gt;&lt;/head&gt;&lt;body&gt;
&lt;p&gt;&lt;span style="font-weight:600;"&gt;No tool selected&lt;/span&gt;&lt;/p&gt;
&lt;p&gt;Select a tool from the list to view details.&lt;/p&gt;
&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
